urls.py

```diff
@@ -55,6 +55,7 @@
     (r'^data/', include('corehq.apps.data_interfaces.urls')),
     (r'^', include(hqwebapp_domain_specific)),
     (r'^case/', include('corehq.apps.hqcase.urls')),
+    (r'^case/', include('corehq.apps.case_search.urls')),
```

Comment:
No reason, really. I guess I was just trying to keep the urls clean (I think case/search is much nicer than case_search/)